### DISCUSSION TITLE
License With Downloadable file

### DISCUSSION THREAD

**Discussion by micheal332001** [boxbilling/boxbilling#798] [General]
It would be great if you could combine the download and license together so that you can sell the download with the license at the same time.

**Comment by yagiz-dev** [comment]
~What license? We've removed all license stuff a month ago. You don't need to buy a license to use BoxBilling anymore, just download the latest release and start using it right away.~

Seems like you're talking about the downloadable products, my bad. Can you turn this into an issue, and not a discussion instead?

**Reply by micheal332001** [reply]
will do
